frontend/public/src/containers/pages/DashboardPage_test.js

```diff
@@ -1,29 +1,22 @@
-/* global SETTINGS: false */
 // @flow
 import { assert } from "chai"
-import moment from "moment"
-import sinon from "sinon"
-import { Formik } from "formik"
+
 
 import DashboardPage, {
   DashboardPage as InnerDashboardPage
 } from "./DashboardPage"
-import { ALERT_TYPE_DANGER, ALERT_TYPE_SUCCESS } from "../../constants"
-import { formatPrettyDateTimeAmPmTz } from "../../lib/util"
-import { shouldIf, isIf } from "../../lib/test_utils"
+
 import IntegrationTestHelper from "../../util/integration_test_helper"
 import { makeCourseRunEnrollment } from "../../factories/course"
 import { makeUser } from "../../factories/user"
-import * as courseApi from "../../lib/courseApi"
 
 describe("DashboardPage", () => {
-  let helper, renderPage, userEnrollments, currentUser, isLinkableStub
+  let helper, renderPage, userEnrollments, currentUser
 
   beforeEach(() => {
     helper = new IntegrationTestHelper()
     userEnrollments = [makeCourseRunEnrollment(), makeCourseRunEnrollment()]
     currentUser = makeUser()
-    isLinkableStub = helper.sandbox.stub(courseApi, "isLinkableCourseRun")
```

Comment:
The only things removed in this PR are
- unused variable declarations
- unused imports
- assignments to variables which were never ready, as in this case.